.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "0.10"
-  - "4.0"
+  - 4
```

Comment:
keep `4.0` since we want to make sure we support that one specifically in addition to 4 latest.